x/globalfee/feechecker/feechecker.go

```diff
@@ -125,106 +116,6 @@ func (fc FeeChecker) CheckTxFee(
 	return feeCoins, priority, nil
 }
 
-// IsBypassMinFeeTx checks whether tx is min fee bypassable.
```

Comment:
please remove FeeChecker's unused fields

Reply:
removed na, maybe this is from the conflict